### PR TITLE
fix: typo in image name

### LINKED ISSUE #288: Typo in image name
See 
https://github.com/containers/ai-lab-recipes/blob/53af15ba0504b05ffa16118cd953a122f66db5b2/CONTRIBUTING.md?plain=1#L53

Also
https://github.com/containers/ai-lab-recipes/blob/53af15ba0504b05ffa16118cd953a122f66db5b2/recipes/natural_language_processing/chatbot/ai-lab.yaml#L18

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #288

### CODE REVIEW

**Review by Gregory-Pereira — DISMISSED** [review]
/lgtm

File: recipes/natural_language_processing/chatbot/ai-lab.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ application:
         - amd64
       ports:
         - 8001
-      image: quay.io/ai-lab/llamacppp-python:latest
+      image: quay.io/ai-lab/llamacpp-python:latest
```
Comment (Gregory-Pereira) [inline recipes/natural_language_processing/chatbot/ai-lab.yaml:18]: `quay.io/ai-lab/llamacpp_python:latest` is correct

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ application:
         - amd64
       ports:
         - 8001
-      image: quay.io/ai-lab/llamacppp-python:latest
+      image: quay.io/ai-lab/llamacpp-python:latest
```
Comment (Gregory-Pereira) [inline CONTRIBUTING.md:53]: `quay.io/ai-lab/llamacpp_python:latest` is correct

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm but swap the images to what you will find in quay: https://quay.io/repository/ai-lab/llamacpp_python

**Comment by rhatdan** [comment]
Replaced with https://github.com/containers/ai-lab-recipes/pull/292

**Pull request closed without merging**